pkg/api/apiloader.go

```diff
@@ -71,14 +71,9 @@ func (a *Apiloader) DeserializeContainerService(contents []byte, validate, isUpd
 		if isAgentPoolOnlyClusterJSON(contents) {
 			log.Info("No masterProfile: interpreting API model as agent pool only")
 			service, _, err := a.LoadContainerServiceForAgentPoolOnlyCluster(contents, version, validate, isUpdate, "", existingContainerService)
-			if service == nil || err != nil {
-				log.Infof("Error returned by LoadContainerServiceForAgentPoolOnlyCluster: %+v", err)
-			}
 			return service, version, err
 		}
-		log.Infof("Error returned by LoadContainerService: %+v", err)
```

Comment:
there is not point in printing this error as info, we are returning the error and throwing an error in the function that calls this function